youtube.js

```diff
@@ -0,0 +1,121 @@
+const divnav = document.createElement('div');
+divnav.id = "navigation"
+container.appendChild(divnav);
```

Comment:
Please use the appropriate tags for each element so instead of a normal div kindly wrap your navigation section with a nav tag